src/test/integration/org/apache/cassandra/testing/TestVersionSupplier.java

```diff
@@ -35,7 +35,7 @@ public class TestVersionSupplier
     Stream<TestVersion> testVersions()
     {
         // By default, we test 2 versions that will exercise oldest and newest supported versions
-        String versions = System.getProperty("cassandra.sidecar.versions_to_test", "4.0,5.0");
+        String versions = System.getProperty("cassandra.sidecar.versions_to_test", "5.1");
```

Comment:
Before we merge, please make this "4.0,5.1" or "4.1,5.1" so we don't lose 4.X testing